components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/Device.kt

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.appservices.fxaclient
+
+data class Device(
+        val id: String,
+        val displayName: String,
+        val deviceType: Type,
+        val pushSubscription: PushSubscription?,
+        val pushEndpointExpired: Boolean,
+        val isCurrentDevice: Boolean,
+        val location: Location,
```

Comment:
Do we expect clients to make any use of this location information?  If not, I feel like perhaps we should not expose it over the API boundary, simply to avoid leakage of any unintended internal details.